From df58fc2f3b4a695c15e318a6a1d4142f07bb76c1 Mon Sep 17 00:00:00 2001
From: yann_dm <yann.montmarin@gmail.com>
Date: Mon, 9 Nov 2020 15:55:21 +0100
Subject: [PATCH] Force qt external OpenGL for compatibility with MacOSX osgqt

---
 src/gui/main.cc.in | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git src/gui/main.cc.in src/gui/main.cc.in
index 855a68b..97672f3 100644
--- src/gui/main.cc.in
+++ src/gui/main.cc.in
@@ -43,7 +43,8 @@ void setupApplication ()
 
 int main(int argc, char *argv[])
 {
-  QApplication::setAttribute(Qt::AA_X11InitThreads);
+
+  QApplication::setAttribute(Qt::AA_UseDesktopOpenGL);
 
   SafeApplication a(argc, argv);
 
-- 
2.17.1

